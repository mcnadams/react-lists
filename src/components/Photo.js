import React from 'react';
import PropTypes from 'prop-types';

function Photo({ photoUrl }) {
  return <img src={photoUrl} style={{ 'width': '20vw' }}/>;
}

Photo.propTypes = {
  photoUrl: PropTypes.string.isRequired
};

export default Photo;
