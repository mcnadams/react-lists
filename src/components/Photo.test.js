import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';

describe('Photo test', () => {
  it('make an img', () => {
    const wrapper = shallow(<Photo photoUrl='./path' />);
    expect(wrapper).toMatchSnapshot();
  });
});
