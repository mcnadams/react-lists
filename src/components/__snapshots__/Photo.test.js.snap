// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo test make an img 1`] = `
<img
  src="./path"
  style={
    Object {
      "width": "20vw",
    }
  }
/>
`;
